third_party/build_deps.sh

```diff
@@ -0,0 +1,271 @@
+#!/bin/bash -xe
+
+ROOT_DIR=$(pwd)
+export INSTALL_PREFIX=${INSTALL_PREFIX:-/usr/local}
+export CC_COMP=${CC_COMP:-gcc}
+export CXX_COMP=${CXX_COMP:-g++}
+echo ${INSTALL_PREFIX}
+echo ${CC_COMP}
+echo ${CXX_COMP}
+echo ${CMAKE_TARGET_ARCH}
+echo ${BUILD_ARCH_OPTION}
+echo ${HOST_ARCH_OPTION}
+echo ${SYSROOT_ARG}
+echo ${WITH_FFMPEG}
+echo ${EXTRA_PROTOBUF_FLAGS}
+echo ${OPENCV_TOOLCHAIN_FILE}
+echo ${EXTRA_FLAC_FLAGS}
+echo ${EXTRA_LIBSND_FLAGS}
+
+#zlib
+pushd third_party/zlib                                                            && \
```

Comment:
I agree that excessive && are, well, excessive.
Or the problem is with setting the variables for the following commands?

Reply:
Done